src/Result.mo

```diff
@@ -67,52 +67,55 @@ public func mapErr<Ok, Error1, Error2>(
   x : Result<Ok, Error1>,
   f : Error1 -> Error2
 ) : Result<Ok, Error2> {
-  switch x {
+  switch(x) {
     case (#err e) (#err (f e));
     case (#ok r) (#ok r);
   }
 };
 
-/// Create a result from an option, including an error value to handle the `null` case.
-/// ```
-/// fromOption(?(x), e) = #ok(x)
-/// fromOption(null, e) = #err(e)
+/// Create a `Result` from an `Option`, including an error value to handle the
+/// `null` case.
+/// ```motoko
+/// fromOption(?x, e) = #ok(x);
+/// fromOption(null, e) = #err(e);
 /// ```
 public func fromOption<R, E>(x : ?R, err : E) : Result<R, E> {
-  switch x {
-    case (? x) {#ok x};
-    case null {#err err};
+  switch(x) {
+    case (?x) (#ok x);
+    case (null) (#err err);
```

Comment:
I'd actually prefer changing the parens to braces elsewhere, again because that's more in the line with other curly-braces languages.